backend/instagram_backend/apps/users/models.py

```diff
@@ -0,0 +1,65 @@
+from django.db import models
+
+
+class UserManager(models.Manager):
+    def create_user(self, email, name, username, password, bio='', profile_picture=None):
+        user = self.model(
+            email=email,
+            name=name,
+            username=username,
+            password=password,
+            bio=bio,
+            profile_picture=profile_picture
+        )
+        user.save(using=self._db)
```

Comment:
unless you are using multiple databases, there is no need to specify which one you are using